### ISSUE TITLE
[Feature Request] Enhancement - Add $Title to Advanced Settings | Renaming Options 

### ISSUE THREAD

**Issue by Dude77777** [mylar3/mylar3#756]
**Describe the request**
As a collector, I want to be able to see the title of individual comics in the name so that I a) can manually search for it and b) that Mylars labelling for new additions is insync with the rest of my collection. Ideally Mylar matches all the tags available in ComicRack over time, but $Title is so far the most important missing one. 

**How did you come up with the idea?**
$Title is a (Top10 / standard) Tag in ComicRack and ComicTagger

**Environment (please complete the following information):**
 - Windows10
 - Mylar v0.5.1

**Additional information**
Add any other information about the request here.

**Comment by evilhero** [comment]
**Short answer:**
Probably not going to happen anytime soon, especially since there are more important things that are on the backburner that need attention. Adding in that kind of parsing functionality, especially given how wide-open the naming conventions are in comics, makes it an extremely difficult aspect to integrate properly without generating false positives and or problems.

**Winded answer:**
The fact that it's a "top 10/standard" is irrelevant  - there is no 'standard' naming convention for comics, and if there was any sort of standard, the standard would most likely be based off of how the overall scene labels their comics - which do not, nor have ever had the issue title in the filenames themselves (that's what the metadata is for).  You might think of it as top 10, someone else might think of it as being top 15, another top 5, someone else not even in the top 20 - it's personal preference.

You're also talking about 2 applications that have literally not been updated in at least 2 years (comictagger), as well as being abandoned (comicrack). The fact that you want Mylar to be able to match up to your various tags from an application that isn't even supported or active any longer, makes it an impossible thing right from the get-go if you're talking about file-name parsing alone. As an example, how is Mylar to know that you have issue year & comic year present within a filename, along with an Issue Title with the Issue Number being first in the filename followed by the a genre tag and then the series title) all seperated by a ' - '. Naming conventions and consistency of users vary widely as opposed to following any sort of specific schema.

Issue Title, while having some benefits from a file-system type of methodology, is not beneficial from a parsing standpoint nor important overall (meaning having it not included doesn't preclude it from matching, but including it could possibly alter the matching aspects depending on where in the filename the issue title is being presented). 

And sure, we can add it in as a File Format option easily enough, however then it wouldn't be able to parse the filename thereafter in order to determine if an issue title is the issue title, or if it's part of another aspect of the filename ( series title, issue number, volume) because your naming convention might be something like ```SomeSuperhero - 001 - v4 - [October 7, 2018]- (Somesuperhero-Issue-Title-with-hyphens-in-it) - scannedbyme - 36p - 5 variant covers.cbr``` Again, you might look at that and say that's not like mine, but I can guarantee you somewhere out there - that's someone's.

Have you thought about removing that aspect from the filename and relying on your comic reader to pull in the proper metadata from within the file, which would therein allow you to search within the given application? (ie, Komga, Ubooquity, or any reader really should be support that option)

**tldr;** I realise that you want your filenames to show the issue title, however adding in that aspect adds in much more complexity to the filename parsing thereafter, which would require a significant amount of time.

**Comment by Dude77777** [comment]
Thank you very much for your detailed answer and explanation evilhero, very much appreciated and I get your point. 
I had never looked into Komga & Ubooquity before, but will check them out now. Cheers
